README.md

```diff
@@ -68,10 +79,17 @@ Sitegeist:
       #
       enabled: true
 
+      #
+      # There are two strategies:
+      # (*) once: will translate the node once the editor switches the language dimension and only then
+      # (*) sync: will translate and sync the node anytime the base language has been edited
+      #
+      strategy: 'once'
```

Comment:
Is that  still true? I thought this is configured in the contentDimensionOptions now

Reply:
I forgot about that, will remove it.